spec/controllers/pieces_controller_spec.rb

```diff
@@ -76,5 +77,56 @@
       end
     end
 
+    context "check?" do
+      it "does not return an error message if a white piece puts the black king in check" do
```

Comment:
for this test to be true shouldn't we show: 1. that king being in check is true 2. error msg is nil or 3. show error message is present

if so, the other tests should follow a similar example